doc/tut1.rst

```diff
@@ -1519,6 +1595,8 @@ variables! For example:
   echo badname
   echo badext
 
+Unlike objects, fields of tuples never need to be exported, they are
```

Comment:
I would just reorder this sentence to something like:

Fields of tuples are always public, they don't need to be explicitly marked to be exported (unlike object fields).